docs/content/what-is-temporal.md

```diff
@@ -18,7 +18,9 @@ Temporal is a scalable and reliable runtime for Reentrant Processes called Tem
 imagePath="/diagrams/temporal-intro.svg"
 imageSize="50"
 title="The Temporal system"
-legendstring="π?Workflow Execution"
+legend={[
+["π","Workflow Execution"],
```

Comment:
are we really sticking to this pi thing

Reply:
Maybe, have to get @dtornow involved there and not something I want to tackle in this particular PR.